spec/controllers/comments_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe CommentsController do
```

Comment:
Typically, you would prefer a request spec over the controller spec. But if I remember correctly, the project only has controller specs, right?

Reply:
Exactly. It has no request specs so far, so we sticked to controller specs.